superset-frontend/spec/javascripts/dashboard/actions/dashboardLayout_spec.js

```diff
@@ -349,24 +349,27 @@ describe('dashboardLayout actions', () => {
       const { getState, dispatch } = setup({
         dashboardLayout: {
           present: {
-            source: { type: ROW_TYPE },
-            destination: { type: ROW_TYPE, children: ['rowChild'] },
-            dragging: { type: CHART_TYPE, meta: { width: 1 } },
-            rowChild: { type: CHART_TYPE, meta: { width: 12 } },
+            source: { id: 'source', type: ROW_TYPE, children: ['dragging'] },
+            destination: {
+              id: 'destination',
+              type: ROW_TYPE,
+              children: ['rowChild'],
+            },
+            dragging: { id: 'dragging', type: CHART_TYPE, meta: { width: 1 } },
+            rowChild: { id: 'rowChild', type: CHART_TYPE, meta: { width: 12 } },
           },
         },
       });
       const dropResult = {
         source: { id: 'source', type: ROW_TYPE },
         destination: { id: 'destination', type: ROW_TYPE },
-        dragging: { id: 'dragging', type: CHART_TYPE },
+        dragging: { id: 'dragging', type: CHART_TYPE, meta: { width: 1 } },
       };
 
       const thunk = handleComponentDrop(dropResult);
       thunk(dispatch, getState);
-      expect(dispatch.getCall(0).args[0].type).toEqual(
-        addWarningToast('').type,
-      );
```

Comment:
Bycatch, this was an invalid test. The invalid input didn't actually test the case it was supposed to test ("drop overflows the destination").

Previously this test passes because both `dispatch.getCall(0).args[0].type` and `addWarningToast('').type` are `undefined`, when they both should be `ADD_TOAST`. The error was surfaced because I removed the `dispatch` wrapper for toast actions.

cc @graceguo-supercat